subsys/bluetooth/services/ias.c

```diff
@@ -0,0 +1,122 @@
+/** @file
+ *  @brief IAS Service sample
+ */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "net/buf.h"
+#include "bluetooth/services/ias.h"
+#include <init.h>
+#include <zephyr.h>
+#include <logging/log.h>
+#include <bluetooth/gatt.h>
+
+#define LOG_LEVEL CONFIG_BT_IAS_LOG_LEVEL
+LOG_MODULE_REGISTER(ias);
+
+#define ALERT_MAX_LEN 1
+#define BT_IAS_ALERT_LVL_NO_ALERT		0
+#define BT_IAS_ALERT_LVL_MILD_ALERT		1
+#define BT_IAS_ALERT_LVL_HIGH_ALERT		2
```

Comment:
Please use spaces for indentation after non-whitespace characters:
```suggestion
#define ALERT_MAX_LEN                    1
#define BT_IAS_ALERT_LVL_NO_ALERT        0
#define BT_IAS_ALERT_LVL_MILD_ALERT      1
#define BT_IAS_ALERT_LVL_HIGH_ALERT      2
```